beacon_node/network/src/service.rs

```diff
@@ -111,19 +109,20 @@ impl<T: BeaconChainTypes> NetworkService<T> {
             propagation_percentage,
         };
 
-        let network_exit = runtime_handle.enter(|| spawn_service(network_service))?;
+        let _ = spawn_service(executor, network_service)?;
```

Comment:
again, why are we dropping this?

Reply:
`spawn_service` returns `()` here as well.